src/main/java/com/github/anopensaucedev/fasterrandom/util/math/random/RandomGeneratorRandom.java

```diff
@@ -85,6 +85,7 @@ public double nextGaussian() {
 	private record Splitter(long seed) implements RandomSplitter {
 		@Override
 		public Random split(int x, int y, int z) {
+			// Warning: MathHelper.hashCode(int, int, int) has been deprecated and will be removed in a future.
```

Comment:
Minecraft uses hashCode, I see no reason to say this because we're just doing what Minecraft does here. If they stop using it, we will too.

Reply:
It is marked as deprecated in the 
[yarn mappings docs](https://maven.fabricmc.net/docs/yarn-1.20.6+build.3/net/minecraft/util/math/MathHelper.html#hashCode(int,int,int))

Reply:
Also, could you please add me to the list of contributors?

Reply:
Yes, but CheckedRandom makes use of hashCode, and `RandomGeneratorRandom` is a drop-in replacement for CheckedRandom. This is just us keeping parity with CheckedRandom. Also most IDE's warn you already about hashCode being deprecated.

Reply:
ok, should I remove the warning comment?